docs/guides/preinstall.md

```diff
@@ -4,77 +4,111 @@ These steps are common to most distros. Please refer to both this page and this
+
+Please, refer to this guide: [Basic setup](https://wiki.t2linux.org/guides/postinstall/).
 
-For other distros, you can download the distro's normal install ISO, but you will have to use an external USB keyboard and mouse for this install process, and you may need a wired internet connection. Additionally, if you later have issues with installing the bootloader (Eg :- GRUB), try booting the ISO with the `efi=noruntime` kernel parameter.
+You can also find detailed info for unsupported debian-based distros [here](https://wiki.t2linux.org/distributions/debian/installation/).
 
 ### Copying the ISO to the USB
 
-User friendly applications for copying the Linux install image to your USB include:
+User-friendly applications for copying the Linux install image to your USB include:
 
 - [USBImager](https://gitlab.com/bztsrc/usbimager/)
-- [balenaEtcher](https://www.balena.io/etcher/) (does collect analytics data).
+- [balenaEtcher](https://www.balena.io/etcher/) (does collect analytics data)
 
-!!! hint "`dd`"
-    If you are familiar with the `dd` tool and the command line, you can use that instead. Use `diskutil list` to find the correct disk, then `sudo dd if=path/to/filename.iso of=/dev/rdiskX bs=1m`. Press `control-T` while it is running to make it show its current progress. **Make sure you select the correct disks.**
+Or you can use the macOS terminal:
 
-While the installation image is being written to the USB, you can [copy Wi-Fi firmware](#copy-wi-fi-firmware), but don't follow the steps after that until it has finished writing the image.
+1. Insert your USB drive.
+2. Open macOS Terminal.
+3. Run `diskutil list` to list all the drives.
+4. Look up for your USB thumb which appears as an **external, physical** drive labelled `/dev/diskX` where `X` is a single number (e.g. /dev/disk**2**).
+5. Run `sudo diskutil unmountDisk /dev/diskX` to unmount the disk.
+6. Run `sudo dd if=path/to/linux.iso of=/dev/rdiskX bs=1m` to start writing. To fetch the iso path easily you can just drag and drop the .iso into the Terminal.
+7. Now you may press `control-T` to make it show how many KBs it has written so far.
+
+While the installation image is being written to the USB, you can skip to [Copy Wi-Fi firmware](#copy-wi-fi-firmware); but don't follow the steps after it, wait until the ISO has been written to disk.
 
 ## Copy Wi-Fi firmware
 
-Linux's Wi-Fi driver uses the same Wi-Fi firmware files as macOS, so we copy these files from macOS to the EFI partition where Linux can access these files and then install them.
+Linux's Wi-Fi driver uses the same Wi-Fi firmware files as macOS, so we copy these files from macOS to the EFI partition where Linux can access and eventually install them.
+
+[Follow here](https://wiki.t2linux.org/guides/wifi-bluetooth/#on-macos) the instructions for the **first part done in macOS** and come back to this page.
```

Comment:
Can we simplify this line a bit? Follow the first part to be done in macOS of [this guide], maybe something like this.

Reply:
It sounds a lot better now: [12b7aa0](https://github.com/gianmaticus/wiki/commit/12b7aa0d90363d5052a341a5bb10283531e9d9e2)